subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[console]]
+== The Gradle Console
+
+NOTE: This feature is a work in progress.
```

Comment:
The feature I am thinking of is "improved console behavior and logging". Even after 4.0 I assume we won't be done. That's my thinking behind it. But I'd agree that a user probably won't have that intricate knowledge and thinks it is either a bug or a missing feature of existing functionality. Will remove it.